### DISCUSSION TITLE
How to check if X JS Framework is in use?

### DISCUSSION THREAD

**Discussion by JLAcostaEC** [JanDeDobbeleer/oh-my-posh#5786] [Troubleshoot] [answered]
Hello!

I'm creating a personal template, and I want to know what is the best way to check for X JS Framework (Currently: SvelteJS)

The way I was able to achieve this was using: `{{ readFile "svelte.config.js" }}`, but I don't need to read it, I just want to check if the file exists. (Same for other frameworks like Qwik or Astro)

Thanks

**Comment by JanDeDobbeleer** [comment, marked as answer]
```json
{
  "properties": {
    "display_mode": "files",
    "extensions": [
      "svelte.config.js"
    ],
    "home_enabled": false,
    "fetch_version": false
  },
  "template": " svelte ",
  "foreground": "p:green",
  "background": "transparent",
  "type": "node",
  "style": "plain"
}
```
